R/utilities-simulation.R

```diff
@@ -84,8 +84,6 @@ saveSimulation <- function(simulation, filePath) {
 
 #' @title
 #'
-#' DEPRECATED
```

Comment:
Lets not deprecate it any more, I think it makes sense to have the two versions as they now return different types.